### PR TITLE
adding chris and daniel as codeowners so they can review PRs too

### PR DESCRIPTION
**Pull request by estib**
#### Contribution Type

Repo Management

#### Description

Adds Chris and Daniel as codeowners so they can also review prs

#### Value Add / Motivation

Move faster!

#### Sources

N/A

#### Testing Strategy

...hopefully it works?

### CODE REVIEW

**Review by ckelner — APPROVED** [review]
LGTM

//cc @ccwells you are the only maintainer for https://github.com/orgs/DataDog/teams/dpn - probably makes sense to at least add @chrisluekermann to that team (maybe Daniel if he's going to be working on DPN related things). Steve and I both tried to add them but we don't have rights to do so, so taking this PR approach for this one repo ;)

**Comment by estib** [comment]
can always adjust if Charlie adds them to that group. i'll merge for now to unblock things.

**Pull request merged**
